packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -57,56 +61,14 @@ export interface HttpRouteMatch {
    * @default - do not match on HTTP2 request protocol
    */
   readonly protocol?: HttpRouteProtocol;
-}
-
-/**
- * Supported values for matching routes based on the HTTP request method
- */
-export enum HttpRouteMatchMethod {
-  /**
-   * GET request
-   */
-  GET = 'GET',
-
-  /**
-   * HEAD request
-   */
-  HEAD = 'HEAD',
-
-  /**
-   * POST request
-   */
-  POST = 'POST',
-
-  /**
-   * PUT request
-   */
-  PUT = 'PUT',
-
-  /**
-   * DELETE request
-   */
-  DELETE = 'DELETE',
-
-  /**
-   * CONNECT request
-   */
-  CONNECT = 'CONNECT',
-
-  /**
-   * OPTIONS request
-   */
-  OPTIONS = 'OPTIONS',
 
   /**
-   * TRACE request
-   */
-  TRACE = 'TRACE',
-
-  /**
-   * PATCH request
+   * The query parameter to match on.
+   * All specified query parameter must match for the route to match.
```

Comment:
@dfezzie & @rishijatia - Please confirm if this statement is accurate.

Reply:
Yes, query parameters (and other matches) are all AND conditions. So this sounds accurate to me.

Reply:
Thanks for the confirmation!